source/qml/SettingsPage.qml

```diff
@@ -875,13 +895,13 @@ ScrollView {
 
                         Label {
                             text: qsTr("Baud rate")
-                            visible: (radioModelCombo.currentIndex !== 2) // smogradio only supports 115200
+                            visible: (radioModelCombo.currentIndex !== 3) // smogradio only supports 115200
                         }
 
                         ComboBox {
                             id: radioBaudRate
-                            model: ['4800','9600']
-                            visible: (radioModelCombo.currentIndex !== 2) // smogradio only supports 115200
+                            model: ['4800','9600','19200']
```

Comment:
19200 will show up for every radio besides smogradio.
FT-817 and TS-2000 have 4800/9600 "wired in", setting 19200 here will result in those radios using 9600 internally but showing up as 19200 on the UI. To solve this problem, you'll need to either:
- deal with 19200 for those radios (if they support it, I can't recall right now)
- change the `model` based on `radioModelCombo.currentIndex`